<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPM">
    <property name="title">
     <string>PM</string>
    </property>
    <addaction name="actionRandomGeneration"/>
    <addaction name="separator"/>
    <addaction name="actionGenerateTrainingData"/>
    <addaction name="separator"/>
    <addaction name="actionGeneratePredictedData"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPM"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRandomGeneration">
   <property name="text">
    <string>Random Generation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGenerateTrainingData">
   <property name="text">
    <string>Generate Training Data</string>
   </property>
  </action>
  <action name="actionGenerateLocalTrainingData">
   <property name="text">
    <string>Generate Local Training Data</string>
   </property>
  </action>
  <action name="actionGeneratePredictedData">
   <property name="text">
    <string>Generate Predicted Data</string>
   </property>
  </action>
  <action name="actionGenerateTrainingDataTrunk">
   <property name="text">
    <string>Generate Training Data (Trunk)</string>
   </property>
  </action>
  <action name="actionGeneratePredictedDataTrunk">
   <property name="text">
    <string>Generate Predicted Data (Trunk)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
